docs/config.c

```diff
@@ -97,7 +97,7 @@
 /*
 ** .pp
 ** The default file in which to save aliases created by the
-** \fC$<create-alias>\fP function. Entries added to this file are
+** \f[CR]$<create-alias>\fP function. Entries added to this file are
```

Comment:
In the Linux man-pages project we completely avoid both \fC and \f[CR].  We use either italics or bold.

Rationale: most of the time, the manual page is read most of the time in the terminal, where everything is monospaced, so marking some identifier as Courier (monospaced) doesn't help the reader at all.  It helps on the PDF, but few people read manual pages as PDF these days.

Reply:
@flatcap do you have any opinions on whether to keep the `CR` or change to something else? Maybe bold **and** italic? What do you use on kernel's documentation in this case @alejandro-colomar? 

I have no strong opinions on this one and used `CR` to keep the same behaviour as before but in a compatible way (both troff and groff support it). After all, I think this predates neomutt by a lot (6cd98f2bb8fbe0b75bf2de3338e034db1de9fded) so I think it was mainly not touched for a while and just existed on the codebase.

Reply:
> most of the time, the manual page is read in the terminal, where everything is monospaced

Oh yeah :-)

> do you have any opinions on whether to keep the `CR` or change to something else?

I think bold would be a reasonable choice for highlighting keywords, functions, etc.
(but I don't feel  too strongly about it).

Reply:
there are actually rules for that: https://man7.org/linux/man-pages/man7/groff_man_style.7.html, section "Font style macros"

Reply:
Indeed, we have style rules for this in groff_man_style(7).  Here's the rules that @ossilator mentions:

```
       .B [text]
              Set text in bold.  If no argument is given, a one‐line input trap
              is planted; text on the next line, which can be further formatted
              with a macro, is set in bold.

              Use  bold  for  literal portions of syntax synopses, for command‐
              line options in running text, and for  literals  that  are  major
              topics  of  the  subject under discussion; for example, this page
              uses bold for macro, string, and register names.  In  an  .EX/.EE
              example  of  interactive  I/O (such as a shell session), set only
              user input in bold.

       .I [text]
              Set text in an italic or oblique face.  If no argument is  given,
              a  one‐line  input  trap is planted; text on the next line, which
              can be further formatted with a macro, is set  in  an  italic  or
              oblique face.

              Use  italics  for file and path names, for environment variables,
              for C data types, for enumeration or preprocessor constants in C,
              for variant (user‐replaceable) portions of syntax  synopses,  for
              the  first  occurrence (only) of a technical concept being intro‐
              duced, for names of journals and of literary works longer than an
              article, and anywhere a parameter requiring  replacement  by  the
              user  is  encountered.   An  exception involves variant text in a
              context already typeset in italics, such as file  or  path  names
              with replaceable components; in such cases, follow the convention
              of  mathematical typography: set the file or path name in italics
              as usual but use roman for the variant part (see .IR and .RI  be‐
              low),  and  italics again in running roman text when referring to
              the variant material.
```

A rough TL;DR would be: variable names and placeholders in italics, and literals and constants in bold.  Examples of commands, expressions, etc., in italics.  Use of bold should be minimal (think of it like uppercase; abusing it is considered rude).

Reply:
OK, italics it is.

Reply:
I was about to type that I've changed everything to italics but looking at the diff, it didn't see right. So I was going to go through everything to see were bold should be used. @alejandro-colomar was faster than me. I'll add his suggestions later - not sure if today because I need to finish updating my last year thesis with the reviewer notes and submit before 2025 ;-)

Reply:
Heh!  Have a lovely new year!  If your thesis allows...  :-)